Android/GreenCircle/app/src/main/java/com/greencircle/framework/views/fragments/services/ServicesFragment.kt

```diff
@@ -90,7 +97,10 @@ class ServicesFragment : Fragment() {
         bundle.putBoolean("fotovoltaics", fotoVoltaicSwitch.isChecked)
         bundle.putBoolean("solarHeaters", solarHeaterSwitch.isChecked)
 
-        // Change to the next view
-        // val fragment = ServicesFragment()
+//        val uploadDocumentsFragment = UploadDocumentsFragment()
```

Comment:
Why is this code commented? If it is not needed, please remove it

Reply:
I dont know the company creation flow, so lefting it commented until it is clear